aws/route53/resources.py

```diff
@@ -0,0 +1,32 @@
+from conftest import botocore_client
+
+
+def zones():
+    """
+    https://botocore.amazonaws.com/v1/documentation/api/latest/reference/services/route53.html#Route53.Client.list_hosted_zones
+    """
+    return (
+        botocore_client.get("route53", "list_hosted_zones", [], {})
+        .extract_key("HostedZones")
+        .flatten()
+        .values()
+    )
+
+
+def cnames():
+    records = []
+    zone_list = zones()
+
+    for zone in zone_list:
+        zone_id = zone["Id"].split("/")[2]
+        zone_records = (
+            botocore_client.get(
```

Comment:
This will iterate through all regions and profiles, which can cause problems. Since Route 53 is a global service, regions aren't a problem here, and we apparently don't use multiple profiles either. The fix is to pass at least the `profiles` parameter as Bob did in #334.

Reply:
re: regions iteration - This won't iterate through regions since https://github.com/mozilla/frost/pull/343/files#diff-5fc95b7e7fea5fe0334c6dffe5abedb8R14 was added in this PR.

Reply:
Is the profile bit needed since the route53 api calls do not iterate through regions?

Reply:
I think it's ok to not include it. @smarnach is correct in that there would be a problem if multiple profiles are used, but this is true with a number of our tests. Current plan is to deprecate allowing multiple AWS profiles in a single run.